### PR TITLE
Fixed ansible state for create update data flow

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
https://github.com/open-cluster-management/backlog/issues/12759

https://github.com/open-cluster-management/backlog/issues/12758

### CODE REVIEW

File: src-web/nls/platform.properties [hunk 1]
```diff
@@ -177,8 +177,9 @@ ansible.scheduleMode.disabled.title = Disable automation
 ansible.scheduleMode.disabled.info = Automation does not run until this setting is updated.
 ansible.unsaved.data = You made changes to the configuration. Do you want to save your policy automation? Click the Cancel button to close.
 ansible.loading.info = Loading policy violation automation data...
-ansible.create.success = New policy violation automation ${0} is successfully created on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
-ansible.edit.success = Existing policy violation automation ${0} is successfully updated on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
+ansible.manual.success = The automation has been manually triggered successfully. Please see history tabe to check ansible history information.
```
Comment (ycao56) [inline src-web/nls/platform.properties:180]: typo tab`e`

Comment (ycao56) [reply src-web/nls/platform.properties:180]: @dockerymick can you review the strings here?

Comment (dockerymick) [reply src-web/nls/platform.properties:180]: @ycao thanks for pulling me in. 

I suggest the following updates:

* Can we use only one period in line 179?
* The automation has been manually initiated successfully. View the History tab to check the ansible history information.

File: src-web/nls/platform.properties [hunk 2]
```diff
@@ -177,8 +177,9 @@ ansible.scheduleMode.disabled.title = Disable automation
...
+ansible.once.success = Policy automation has been set to once mode. The automation will be automatically triggered once when policy is in NonCompliant status. Please see history tab to check ansible history information.
```
Comment (dockerymick) [inline src-web/nls/platform.properties:181]: Policy automation is set to once mode. The automation is automatically initiated once when the policy is in a NonCompliant state. View the History tab to check the ansible history information.

File: tests/jest/config/platform-properties.json [hunk 3]
```diff
@@ -149,8 +149,9 @@
   "ansible.scheduleMode.disabled.info": "Automation does not run until this setting is updated.",
   "ansible.unsaved.data": "You made changes to the configuration. Do you want to save your policy automation? Click the Cancel button to close.",
   "ansible.loading.info": "Loading policy violation automation data...",
-  "ansible.create.success": "New policy violation automation ${0} is successfully created on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.",
-  "ansible.edit.success": "Existing policy violation automation ${0} is successfully updated on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.",
+  "ansible.manual.success": "The automation has been manually triggered successfully. Please see history tabe to check ansible history information.",
```
Comment (dockerymick) [inline tests/jest/config/platform-properties.json:152]: Suggestions mentioned recently should also be applied to this file if accepted

**Review by dockerymick — COMMENTED** [review]
Thanks @ycao56 for pulling me in. I made a few comments

File: src-web/nls/platform.properties [hunk 4]
```diff
@@ -176,9 +176,10 @@ ansible.scheduleMode.once.info = When this policy is violated, the automation ru
...
 ansible.unsaved.data = You made changes to the configuration. Do you want to save your policy automation? Click the Cancel button to close.
-ansible.loading.info = Loading policy violation automation data...
-ansible.create.success = New policy violation automation ${0} is successfully created on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
-ansible.edit.success = Existing policy violation automation ${0} is successfully updated on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
+ansible.loading.info = Loading policy violation automation data.
+ansible.manual.success = The automation has been manually initiated successfully. View the History tab to check the ansible history information.
+ansible.once.success = Policy automation is set to once mode. The automation is automatically initiated once when the policy is in a NonCompliant state. View the History tab to check the ansible history information.
+ansible.disabled.success = Suggestions mentioned recently should also be applied to this file if accepted.
```
Comment (dockerymick) [inline src-web/nls/platform.properties:182]: This was in my last comment, I think this should be something different right?

Comment (ChunxiAlexLuo) [reply src-web/nls/platform.properties:182]: @dockerymick 
Screen Shot 2021-05-17 at 1 11 10 PM

**Review by ycao56 — APPROVED** [review]

File: src-web/nls/platform.properties (hunk 4 above)
Comment (dockerymick) [reply src-web/nls/platform.properties]: I suggest updating that to read:

The automation is not automatically initiated when it is set to disable mode.

Comment (dockerymick) [reply src-web/nls/platform.properties]: @ChunxiAlexLuo The suggestions weren't applied to the file. Can you avoid using the word "trigger". I recommend that we use initiate instead

Comment (ycao56) [reply src-web/nls/platform.properties]: @dockerymick I think @ChunxiAlexLuo will fix this in his next pr.

**Pull request merged**